test/test_printing_utils.py

```diff
@@ -16,6 +15,15 @@
 from unittest.mock import patch, Mock
 
 
+FOO_COLUMNS = [
```

Comment:
```suggestion
TEST_COLUMNS = [
```
slightly easier to read, IMHO